definitions/variable/industry/tag_chemical_commodities.yaml

```diff
@@ -0,0 +1,23 @@
+- Chemicals Commodity:
```

Comment:
Is there a need for separating the chemicals sub-categories into three different tag-lists?

Reply:
It is not absolutely needed no. I did it to group commodities according to their "position" in the supply chain and to have a separate group for commodities with primary and secondary production. Alternatively, this can of course also be defined in the same scheme that you suggested for iron commodities above.